src/MCPServer/lib/archivematicaMCP.py

```diff
@@ -367,15 +331,15 @@ def _except_hook_log_everything(exc_type, exc_value, exc_traceback):
 if __name__ == "__main__":
 
     # Replace exception handler with one that logs exceptions
-    sys.excepthook = _except_hook_log_everything
+    # sys.excepthook = _except_hook_log_everything
```

Comment:
Should this line be removed?

Reply:
Uhh, probably not! That looks like an accident. I'll revert that change.